projects/core/src/state/utils/entity-loader/entity-loader.action.ts

```diff
@@ -80,7 +81,7 @@ export class EntityFailAction implements EntityLoaderAction {
 export class EntitySuccessAction implements EntityLoaderAction {
   type = ENTITY_SUCCESS_ACTION;
   readonly meta: EntityLoaderMeta;
-  constructor(entityType: string, id: string | string[]) {
+  constructor(entityType: string, id: string | string[], public payload?: any) {
```

Comment:
why we add this `payload`?

Reply:
Hmm, I think we had an idea where to use it, but I don't actually see it being used. @dunqan do you remember maybe?

Reply:
Yup, you are right @znikola, IMO it can stay (because we found, that without it generic EntitySuccessAction is not complete), but I can also add it in another ticket.

Reply:
Ok, I'll just leave it